CMakeLists.txt

```diff
@@ -57,7 +56,7 @@ if(CMAKE_INSTALL_PREFIX STREQUAL "${AliPhysics_SOURCE_DIR}")
 endif()
 
 # Path to additonal modules
-set(CMAKE_MODULE_PATH "${AliPhysics_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH "${AliPhysics_SOURCE_DIR}/cmake" "${AliPhysics_SOURCE_DIR}/OADB")
```

Comment:
As said already, in the CMakeLists.txt closer to the directory `PWGCF/Correlations/DPhi`. This means `PWGCF/Correlations/DPhi/CMakeLists.txt`.

You have to change, as discussed with @mpuccio and @rbertens:

* `ALIROOT_DEPENDENCIES` to add OADB, the new line becoming `set(ALIROOT_DEPENDENCIES ANALYSISalice CORRFW PWGmuon PWGCFCorrelationsBase PWGJETFW OADB)`
* `${AliPhysics_SOURCE_DIR}/OADB` in the `include_directories` statement